content/attacks/posts/2019-01-28-Coincheck.md

```diff
@@ -0,0 +1,37 @@
+
+## Attackers
+
+The perpetrators behind the Coincheck NEM security breach were linked to a Canadian exchange and a Japanese exchange called Zaif, as identified by the BIG Blockchain Intelligence Group Inc. To facilitate the tracking of stolen funds, the NEM development team implemented an automated tagging system. This system ensures that all funds stolen from Coincheck are marked as tainted funds, allowing cryptocurrency exchanges to easily verify whether stolen NEM funds are withdrawn or deposited into regulated trading platforms. Consequently, the hackers are currently left with limited options, unable to convert the stolen NEM to other major cryptocurrencies like Bitcoin and Ethereum due to the immediate alert triggered by the automated tagging system.
+
+Tokyo police made significant progress in the case, leading to the arrest of two men suspected of hacking Coincheck. The investigation focused on tracking a large-scale transaction that occurred on the dark web, where a substantial amount of NEM was exchanged at a 15% discount. Authorities believe that the hackers managed to convert their XEM funds into Bitcoin. On March 11, 2022, the police took two individuals into custody. While details about the detainees are limited, it is known that they are males—one from Osaka Prefecture and the other from Hokkaido. The Osaka resident stands accused of transacting 24 million NEM coins and potentially accessing around 200 accounts. Both individuals were arrested under the provisions of the Punishment of Organized Crimes and Proceeds of Crime Control Act.
+
+## Losses
+
+The attack resulted in the theft of 523 million NEM tokens, valued at approximately $530 million at the time. The stolen funds belonged to Coincheck's users, consisting of both individual traders and institutional investors. The incident significantly impacted Coincheck's reputation and raised concerns about the security practices implemented by cryptocurrency exchanges.
+
+## Timeline
+
+- **January 26, 2019:** Coincheck detected an unauthorized outflow of NEM tokens from their hot wallet, prompting an immediate investigation into the security breach.
+- As a precautionary measure, Coincheck swiftly suspended all withdrawals and deposits of NEM tokens and other cryptocurrencies.
+- **January 27, 2019:** Coincheck publicly confirmed the security breach, acknowledging the substantial loss of NEM tokens and pledging to compensate affected users.
+- Authorities and regulatory bodies initiated investigations into the incident, focusing on identifying the attackers and recovering the stolen funds.
+- Coincheck announced its intention to compensate affected users by repurchasing the stolen NEM tokens at a fixed rate, ensuring reimbursement for each user impacted by the hack.
+- **March 12, 2019:** Coincheck completed the reimbursement process, compensating affected users with Japanese yen in exchange for their lost NEM tokens.
```

Comment:
add the arrest of the alleged suspects to the timeline

Reply:
All done